src/main/java/com/softserveinc/ita/pkuravskyi/pageobjects/SoftServePage.java

```diff
@@ -20,22 +20,24 @@ public SoftServePage selectSideNavBarCategory(String category) {
         var sideNavBarItems = $$x(sideNavBarCategoriesList);
 
         for (var item : sideNavBarItems) {
-            if (!item
-                    .getAttribute("text")
+            var itemAttribute = item.getAttribute("text");
+
+            if (!itemAttribute
```

Comment:
avoid using more than one sequence of methods, move 'substring' into a variable, and chain the methods

item
.getAttribute("text")
.substring(2);